core/src/main/java/org/bitcoinj/core/CheckpointManager.java

```diff
@@ -66,19 +66,18 @@
  * different concept of checkpoints that are used to hard-code the validity of blocks that violate BIP30 (duplicate
  * coinbase transactions). Those "checkpoints" can be found in NetworkParameters.</p>
  *
- * <p>The file format consists of the string "CHECKPOINTS 1", followed by a uint32 containing the number of signatures
+ * <p>The file format is textual, one value per line.
+ * It consists of the magic string "TXT CHECKPOINTS 1", followed by the number of signatures
  * to read. The value may not be larger than 256 (so it could have been a byte but isn't for historical reasons).
```

Comment:
"so it could have been a byte" looks leftover from the binary format.

Reply:
Removed.